config.toml

```diff
@@ -9,3 +8,4 @@ registration_type = "registration_types"
 [params]
 Owner = "USA.gov"
 Referrer_Short_Code = "voteusa"
+lang = "en-US"
```

Comment:
When using translations in Hugo, if the `config.toml` doesn't contain a `lang = "*"` property, it causes a `go panic`. See spf13/hugo#2174 for more information.